include/glow/Graph/PlaceholderBindings.h

```diff
@@ -38,18 +38,12 @@ class Placeholder;
 class PlaceholderBindings final {
 public:
   /// Maps placeholders to the tensors that back them.
-  using PlaceholderMap = std::unordered_map<Placeholder *, Tensor *>;
-
-  /// Maps Placeholder names to Placeholders.
-  using PlaceholderNameMap = std::unordered_map<std::string, Placeholder *>;
+  using PlaceholderMap = std::unordered_map<const Placeholder *, Tensor *>;
```

Comment:
I think that this is fine for now. Keep in mind that LLVM may have more efficient data structures, such as densemap.